.devcontainer/devcontainer.json

```
@@ -3,7 +3,7 @@
     "build": {
       "dockerfile": "Dockerfile",
       "args": {
-        "VARIANT": "1.23-bookworm"
+        "VARIANT": "1.24.1-bookworm"
```

This tag doesn't exist.
```suggestion
        "VARIANT": "1.24-bookworm"
```